docs/admin/release_notes/version_1.5.md

```diff
@@ -0,0 +1,23 @@
+# v1.5 Release Notes
+
+- Add ability to update JSON based compliance via a job and have git integrations.
+- Made custom compliance a boolean to support both JSON or CLI custom compliance types.
+- Add metrics for Golden Config plugin.
+- Supports Nautobot >=1.6.1,<2.0.0.
+
+## v1.5.0 - 2023-08
+
+### Added
+
+- [455](https://github.com/nautobot/nautobot-plugin-golden-config/pull/455) - Add metrics for Golden Config plugin.
+- [485](https://github.com/nautobot/nautobot-plugin-golden-config/pull/485) - Custom compliance for CLI and JSON rules.
+- [487](https://github.com/nautobot/nautobot-plugin-golden-config/pull/487) - Implement native JSON support.
+
+### Changed
+
+- [485](https://github.com/nautobot/nautobot-plugin-golden-config/pull/485) - Changed the behavior of custom compliance to a boolean vs toggle between cli, json, and custom.
+
```

Comment:
would assume once #553 is merged, we need to get details added.

Reply:
I look at the release notes as what has changed, hence you may see the same PR referenced multiple times. #553 removed #506, so the net effect is zero, but welcome to hear other thoughts.

Reply:
okay, fine by this. Was thinking we had already made a release with 506 in it, but doesn't look like we did.